### DISCUSSION TITLE
Why the RGB-NODE has been archived ? what shuold i do  if  i need use this rep?

### DISCUSSION THREAD

**Discussion by SephirothLT** [RGB-WG/rgb#64] [General]
Why the RGB-NODE has been archived ? what shuold i do  if  i need use this rep?

**Comment by dr-orlovsky** [comment]
Duplicate of: https://github.com/orgs/RGB-WG/discussions/59

![Screenshot from 2023-12-20 08-49-26](https://github.com/RGB-WG/rgb/assets/372034/93a41f06-e130-4093-ade6-53622137a199)
